packages/fscomponents/src/components/Carousel/CarouselProps.ts

```diff
@@ -1,12 +1,13 @@
 import { ReactNode } from 'react';
 import { StyleProp, ViewStyle } from 'react-native';
+import Swiper from 'react-native-swiper';
 
 // TODO: Unify these web/native options and clean up these types ...a lot
 export interface CarouselProps {
   currentPageIndicatorColor?: string;
   height?: number;
   loop?: boolean;
-  nativeOptions?: any;
+  nativeOptions?: Swiper;
```

Comment:
I think this needs to be `SwiperProperties` from `@types/react-native-swiper`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native-swiper/index.d.ts#L32

Using `Swiper` will expect an instance of the component instead.